galaxy_ng/app/__init__.py

```diff
@@ -23,59 +27,152 @@ def ready(self):
+            "summary_fields": common_summary_fields + ("version",),
+            "reverse_name": "content-ansible/collection_versions-detail"
+        },
+        pulp_ansible_models.Collection: {
+            "summary_fields": common_summary_fields + ("namespace",),
+            "reverse_name": "ansible/collections-detail"
+        },
+        pulp_ansible_models.CollectionRemote: {
+            "summary_fields": common_summary_fields,
+            "reverse_name": "remotes-ansible/collection-detail"
+        },
+        pulp_ansible_models.AnsibleRepository: {
+            "summary_fields": common_summary_fields,
+            "reverse_name": "repositories-ansible/ansible-detail"
+        },
+        galaxy_collectionimport_models.CollectionImport: {
+            "summary_fields": common_summary_fields + ("version",),
+            "reverse_name": "galaxy:api:v3:collection-imports-detail"
+        },
+        galaxy_namespace_models.NamespaceLink: {
+            "summary_fields": common_summary_fields,
+        },
+        galaxy_namespace_models.Namespace: {
+            "summary_fields": common_summary_fields,
+            "reverse_name": "galaxy:api:v3:namespaces-detail",
+            "reverse_args": ("name",)
+        },
+        galaxy_container_models.ContainerDistroReadme: {
+            "summary_fields": ("pk",),
+        },
+        galaxy_container_models.ContainerNamespace: {
+            "summary_fields": common_summary_fields,
+            "reverse_name": "pulp_container/namespaces-detail"
+        },
+        galaxy_container_models.ContainerRegistryRemote: {
+            "summary_fields": common_summary_fields,
+            "reverse_name": "galaxy_ng/registry-remote-detail"
+        },
+        galaxy_container_models.ContainerRegistryRepos: {
+            "summary_fields": ("pk",),
+        },
+        pulp_container_models.ContainerDistribution: {
+            "summary_fields": common_summary_fields,
+            "reverse_name": "distributions-container/container-detail"
+        },
+        pulp_container_models.ContainerNamespace: {
+            "summary_fields": common_summary_fields,
+            "reverse_name": "pulp_container/namespaces-detail"
+        },
+        pulp_container_models.ContainerRepository: {
+            "summary_fields": common_summary_fields,
+            # "reverse_name": "repositories-container/container-detail"
+            "reverse_name": "galaxy:api:v3:container-repository-detail",
+            "reverse_args": ("base_path",)
+        },
+        dab_rbac_models.RoleUserAssignment: {
+            "reverse_name": "galaxy:api:ui_v2:roleuserassignment-detail"
+        },
+        dab_rbac_models.RoleTeamAssignment: {
+            "reverse_name": "galaxy:api:ui_v2:roleteamassignment-detail"
+        },
+        dab_rbac_models.RoleDefinition: {
+            "reverse_name": "galaxy:api:ui_v2:roledefinition-detail"
+        },
+        org_team_models.Team: {"reverse_name": "galaxy:api:ui_v2:team-detail"},
```

Comment:
These are the only ones I'm particularly qualified to talk about. Looks fine to me.